src/widgets/mentors/ui/mentors.tsx

```diff
@@ -1,36 +1,40 @@
+import classNames from 'classnames/bind';
 import { LINKS } from '@/app/const';
 import mentorImg from '@/shared/assets/mentors-wanted.webp';
 import { Image } from '@/shared/ui/image';
 import { LinkCustom } from '@/shared/ui/link-custom';
 import { Paragraph } from '@/shared/ui/paragraph';
 import { WidgetTitle } from '@/shared/ui/widget-title';
 
-import './mentors.scss';
+import styles from './mentors.module.scss';
+
+const cx = classNames.bind(styles);
 
 export const Mentors = () => {
   return (
-    <section className="mentors container" id="mentors-wanted">
-      <div className="mentors content column-2">
-        <div className="mentors-info">
+    <section className={cx('container', 'mentors-container')} id="mentors-wanted">
+      <article className={cx('content', 'column-2', 'mentors-content')}>
+        <div className={cx('mentors-info')}>
           <WidgetTitle size="large" mods="lines">Mentors wanted!</WidgetTitle>
-          <Paragraph fontSize="large" className="paragraph">
+          <Paragraph fontSize="large" className={cx('paragraph')}>
```

Comment:
if you apply my suggestion above, this could also be removed

```suggestion
          <Paragraph fontSize="large">
```

Reply:
Done